specification/maps/data-plane/Creator/preview/2.0/wfs.json

```diff
@@ -734,6 +734,156 @@
+              },
+              {
+                "value": "LineString",
+                "description": "`GeoJSON LineString` geometry.",
+                "name": "GeoJsonLineString"
+              },
+              {
+                "value": "MultiLineString",
+                "description": "`GeoJSON MultiLineString` geometry.",
+                "name": "GeoJsonMultiLineString"
+              },
+              {
+                "value": "Polygon",
+                "description": "`GeoJSON Polygon` geometry.",
+                "name": "GeoJsonPolygon"
+              },
+              {
+                "value": "MultiPolygon",
+                "description": "`GeoJSON MultiPolygon` geometry.",
+                "name": "GeoJsonMultiPolygon"
+              },
+              {
+                "value": "GeometryCollection",
+                "description": "`GeoJSON GeometryCollection` geometry.",
+                "name": "GeoJsonGeometryCollection"
+              },
+              {
+                "value": "Feature",
+                "description": "`GeoJSON Feature` object.",
+                "name": "GeoJsonFeature"
+              },
+              {
+                "value": "FeatureCollection",
+                "description": "`GeoJSON FeatureCollection` object.",
+                "name": "GeoJsonFeatureCollection"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "GeoJsonFeatureData": {
+      "type": "object",
+      "properties": {
+        "geometry": {
+          "$ref": "#/definitions/GeoJsonGeometry"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties can contain any additional metadata about the `Feature`. Value can be any JSON object or a JSON null value",
+          "type": "object"
+        },
+        "id": {
+          "description": "Identifier for the feature.",
+          "type": "string"
+        },
+        "featureType": {
+          "description": "The type of the feature. The value depends on the data model the current feature is part of. Some data models may have an empty value.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "geometry"
+      ]
+    },
+    "GeoJsonGeometry": {
+      "description": "A valid `GeoJSON` geometry object. The type must be one of the seven valid GeoJSON geometry types - Point, MultiPoint, LineString, MultiLineString, Polygon, MultiPolygon and GeometryCollection. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.1) for details.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/GeoJsonObject"
+        }
+      ]
+    },
+    "GeoJsonFeatureCollection": {
+      "description": "A valid `GeoJSON FeatureCollection` object type. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.3) for details.",
+      "x-ms-discriminator-value": "FeatureCollection",
```

Comment:
This has the same value as `ExtendedGeoJsonFeatureCollection` which is invalid (all discriminator values must be unique).  Can the value for `ExtendedGeoJsonFeatureCollection` get changed to something like `ExtendedFeatureCollection` (assuming that's unique).

Reply:
We have written a tool to fix this issue. There is no need to modify the content, so I closed it.

Reply:
We have written a tool to fix it.There is no need to modify the content, so I closed this pr.

Reply:
Does this mean that the duplicate discriminator values will be fixed in some other PR?

Reply:
@jhendrixMSFT the duplication was added in this PR to work around an error in doc generation. The main branch shouldn't contain it. 

`
$ find . | grep json | xargs grep x-ms-discriminator-value
./Common/preview/1.0/common.json:      "x-ms-discriminator-value": "Point",
./Common/preview/1.0/common.json:      "x-ms-discriminator-value": "MultiPoint",
./Common/preview/1.0/common.json:      "x-ms-discriminator-value": "LineString",
./Common/preview/1.0/common.json:      "x-ms-discriminator-value": "MultiLineString",
./Common/preview/1.0/common.json:      "x-ms-discriminator-value": "Polygon",
./Common/preview/1.0/common.json:      "x-ms-discriminator-value": "MultiPolygon",
./Common/preview/1.0/common.json:      "x-ms-discriminator-value": "GeometryCollection",
./Common/preview/1.0/common.json:      "x-ms-discriminator-value": "Feature",
./Common/preview/1.0/common.json:      "x-ms-discriminator-value": "FeatureCollection",
./Creator/preview/2.0/featurestate.json:      "x-ms-discriminator-value": "boolean",
./Creator/preview/2.0/featurestate.json:      "x-ms-discriminator-value": "number",
./Creator/preview/2.0/featurestate.json:      "x-ms-discriminator-value": "string",
./Creator/preview/2.0/wfs.json:      "x-ms-discriminator-value": "FeatureCollection",

`

Reply:
Please ignore this pr. now it does not intend to merge into the main branch. I have closed it.

Reply:
The duplicates are outside this PR in main now.
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/maps/data-plane/Creator/preview/2.0/wfs.json#L687

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/maps/data-plane/Common/preview/1.0/common.json#L503

Reply:
ExtendedGeoJsonFeatureCollection is composed of GeoJsonFeatureCollection and they both have the same discriminator value which is malformed.  I haven't checked if there are other instances of this.

Reply:
@jhendrixMSFT good catch. I will review and open separate PRs to fix this. Thanks!

Reply:
cc @zhenglaizhang

Reply:
@daniel-rocha, @jhendrixMSFT: unfortunately there is no different discriminator in ExtendedGeoJsonFeatureCollection, this entity is non-standard, it is a geojson feature collection that contains 3 more non-standard geojson properties (ontology/numberReturned/links), so we won't be able to differentiate it based on the type.